.travis.yml

```diff
@@ -1,15 +1,3 @@
+version: ~> 1.0
 language: python
-python:
-    - 2.6
-    - 2.7
-    - 3.2
-    - 3.3
-install:
-    - pip install --upgrade setuptools
-    - pip install --upgrade zc.buildout
-    - buildout bootstrap
-    - ./bin/buildout
-script:
-    - ./bin/test -vvv
-notifications:
-    email: false
+import: zopefoundation/meta:travis-ci-config/minimal.yml
```

Comment:
Hmm, I suppose if there's a package that cannot march in lockstep with the others then you can override the python versions for just that package?

(Getting [check-python-versions](https://github.com/mgedmin/check-python-versions) to support looking up imported travis config files is going to be a pain.)

I'm not sure how I feel about uncoupling the set of Python versions in .travis.yml from those declared in setup.py, tox.ini and the changelog.  (I suppose the few packages that have an appveyor.yml and multilinux scripts are special enough and few enough that we can deal with them individually.)

Reply:
It should be overridable an there should be other config files for packages which have other requirements for the TravisCI configuration.

I just started at the ZF packages which where not touched for the longest time either archiving them or updating them to a more current status.